internal-packages/react-graphql-test-app/src/components/AddPet.tsx

```diff
@@ -1,11 +1,62 @@
-import { useMutation } from '@data-eden/react';
+import { graphql, useMutation } from '@data-eden/react';
```

Comment:
It seems vaguely odd to import this from `/react` package, it isn't really related to react at all is it?

Reply:
It's just a re-export from `athena`, with the idea being that a react app developer should be install `@data-eden/react` and get everything from there.

Reply:
that is a good piece of context. if you are using react just import the react package.